01-JS-DOM-APIs/index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Index</title>
+    <link rel="stylesheet" href="css/style.css">
+
+  </head>
+  <body onload="fade();">
+    <section class="content">
+      <div class="search">
+        <label for="keyword-repositories">Search repositories: </label>
+        <input type="text" name="keyword-repositories" id="keyword-repositories">
+        <button type="button" id="button-get-repositories">Get repositories</button>
+        <br/>
+        <span id="search-status"></span>
+      </div>
+      <div class="results">
+        <section class="hidden" id="fade-in">
+          <p id="status">Hello world</p>
+          <ul id="repositories-list"></ul>
+        </section>
+      </div>
+    </section>
+
+    <script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js"></script>
+    <script>window.jQuery || document.write('<script src="js/vendor/jquery-{{JQUERY_VERSION}}.min.js"><\/script>')</script>

```

Comment:
oh, I used a [HTML5 boilerplate I found on topic0](https://raw.githubusercontent.com/h5bp/html5-boilerplate/master/src/index.html)... I'll just delete that then